### PR TITLE
UCR pagination

### PR DESCRIPTION
**Pull request by nickpell**
Planning on handling querying for total records and the total row in a separate PR.

@czue @NoahCarnahan

### CODE REVIEW

**Comment by nickpell** [comment]
:white_check_mark:

File: corehq/apps/userreports/reports/view.py [hunk 1]
```diff
@@ -285,11 +285,8 @@ def get_ajax(self, request):
                 'warning': msg
             })
 
-        # todo: this is ghetto pagination - still doing a lot of work in the database
```
Comment (czue) [inline corehq/apps/userreports/reports/view.py]: :raised_hands:

**Comment by czue** [comment]
looks good. any QA concerns?

**Comment by NoahCarnahan** [comment]
:+1:

**Comment by nickpell** [comment]
> looks good. any QA concerns?

Tested on staging, and pagination is handled correctly.  I feel ready to merge.

**Pull request merged**